### ISSUE TITLE
Router Link in ListItem

### ISSUE THREAD

**Issue by webbygregg** [jamesmfriedman/rmwc#176]
I am trying to use a link in a list item by doing this

```
<ListItem tag={Link} to="/">
        <ListItemGraphic>home</ListItemGraphic>
        <ListItemText>Home</ListItemText>
 </ListItem>
```

But this is returning an error: React does not recognize the `elementRef` prop on a DOM element.

Am i doing this incorrectly?

**Comment by jamesmfriedman** [comment]
This is a regression thats popped up since 1.4. What you have should work, I'll log this as a bug and see about getting it fixed.

**Comment by webbygregg** [comment]
Oh right, thank you.

**Comment by jamesmfriedman** [comment]
Fixed with 1.5
